conformance/scenarios/httpgateway/httpgateway_test.go

```diff
@@ -0,0 +1,141 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package httpgateway
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"sigs.k8s.io/gateway-api/apis/v1alpha1"
+	"sigs.k8s.io/gateway-api/conformance/utils"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/klog"
+)
+
+func TestHttpGateway(t *testing.T) {
+	// Test Setup
+	var err error
+	utils.KubeClient, utils.GWClient, err = utils.LoadClientset()
+	if err != nil {
+		klog.Fatalf("error loading Kubernetes clientsets: %v", err)
+	}
+	err = utils.DynamicApply(utils.DynamicParams{Path: "../../../config/crd/bases"})
```

Comment:
applying CRDs seems like it may not be safe. Maybe fine for now and a flag in the future could be useful?

Reply:
Apparently I'm not doing that in the godog test setup. That feels more appropriate since we're relying on the cluster these tests run in to have a GatewayClass resource and corresponding controller preinstalled.